app/uk/gov/hmrc/thirdpartyapplication/models/ApiPlatformEvents.scala

```diff
@@ -38,97 +38,80 @@ object EventType extends Enumeration {
-    oldRedirectUris: String,
-    newRedirectUris: String
-  ) extends ApplicationEvent {
-  override val eventType: EventType.Value = EventType.REDIRECT_URIS_UPDATED
-}
-
-case class ApiSubscribedEvent(
-    override val id: EventId,
-    override val applicationId: String,
-    override val eventDateTime: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC),
-    override val actor: Actor,
-    context: String,
-    version: String
-  ) extends ApplicationEvent {
-  override val eventType: EventType.Value = EventType.API_SUBSCRIBED
-}
-
-case class ApiUnsubscribedEvent(
-    override val id: EventId,
-    override val applicationId: String,
-    override val eventDateTime: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC),
-    override val actor: Actor,
-    context: String,
-    version: String
-  ) extends ApplicationEvent {
-  override val eventType: EventType.Value = EventType.API_UNSUBSCRIBED
-}
+case class TeamMemberAddedEvent(id: EventId,
+                                applicationId: String,
+                                eventDateTime: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC),
+                                actor: OldActor,
+                                teamMemberEmail: String,
+                                teamMemberRole: String) extends ApplicationEvent
+
+case class TeamMemberRemovedEvent(id: EventId,
+                                  applicationId: String,
+                                  eventDateTime: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC),
+                                  actor: OldActor,
+                                  teamMemberEmail: String,
+                                  teamMemberRole: String) extends ApplicationEvent
+
+case class ClientSecretAddedEvent(id: EventId,
+                                  applicationId: String,
+                                  eventDateTime: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC),
+                                  actor: OldActor,
+                                  clientSecretId: String) extends ApplicationEvent
+
+case class ClientSecretRemovedEvent(id: EventId,
+                                    applicationId: String,
+                                    eventDateTime: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC),
+                                    actor: OldActor,
+                                    clientSecretId: String) extends ApplicationEvent
+
+
+// case class PpnsCallBackUriUpdatedEvent(id: EventId,
+//                                        applicationId: String,
+//                                        eventDateTime: LocalDateTime = LocalDateTime.now(ZoneOffset.UTC),
+//                                        actor: OldActor,
+//                                        boxId: String,
+//                                        boxName: String,
+//                                        oldCallbackUrl: String,
+//                                        newCallbackUrl: String) extends ApplicationEvent
```

Comment:
Is this commented out class for future use?

Reply:
No, it shouldn't be there I'll remove it